### ISSUE TITLE
E475: Invalid argument: function <SNR>91_add_methods[2]..<SNR>91_repo_dir

### ISSUE THREAD

**Issue by kutsan** [tpope/vim-fugitive#1371]
This is what I get when I launch vim. I don't have any configuration about this plugin.

```vim
Error detected while processing function <SNR>91_add_methods[2]..<SNR>91_function:
line    1:
E475: Invalid argument: function <SNR>91_add_methods[2]..<SNR>91_repo_dir
Press ENTER or type command to continue
Error detected while processing function <SNR>91_add_methods[2]..<SNR>91_function:
line    1:
E475: Invalid argument: function <SNR>91_add_methods[2]..<SNR>91_repo_prepare
Press ENTER or type command to continue
Error detected while processing function <SNR>91_add_methods[2]..<SNR>91_function:
line    1:
E475: Invalid argument: function <SNR>91_add_methods[2]..<SNR>91_repo_superglob
Press ENTER or type command to continue

Error detected while processing function <SNR>91_add_methods[2]..<SNR>91_function:
line    1:
E475: Invalid argument: function <SNR>91_add_methods[2]..<SNR>91_repo_config
Press ENTER or type command to continue
```

![image](https://user-images.githubusercontent.com/10108377/66331677-c214dc00-e93b-11e9-94ab-c38d57c45a60.png)

**Comment by jsilasbailey** [comment]
Having the same issue after updating to the most recent commit https://github.com/tpope/vim-fugitive/commit/28abadcb847d2026b76f169a147d69ceec8e11a9

**Comment by tpope** [comment]
Fixed by #1373
